hack/kubearchive-install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Install KubeArchive by itself. Used by quick install and also to update existing
+# KubeArchive installations.
+
+set -o errexit
+set -o xtrace
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd ${SCRIPT_DIR}/..
+
+PODS=$(kubectl -n kubearchive get pods | grep -E -v "NAME|No resources|apiserversource" |& awk '{print $1}')
+
+YAML=$(mktemp --suffix=.yaml -t kubearchive-XXX)
+bash cmd/operator/generate.sh
+helm template kubearchive charts/kubearchive -n kubearchive \
+    --include-crds \
+    --set "global.production=true" > /tmp/kubearchive-not-resolved.yaml
+ko resolve -f /tmp/kubearchive-not-resolved.yaml --base-import-paths > ${YAML}
+kubectl apply -n kubearchive -f ${YAML}
+
+rm -f ${YAML}
+
+kubectl -n kubearchive rollout status deployment --timeout=90s
+
+# Wait for all the existing pods to terminate.
```

Comment:
Since the script will be waiting here, you should have an echo so anyone watching knows what is happening.

```suggestion
echo "Waiting for all the existing pods to terminate."
```

Reply:
The command is echoed here so the user will know what is going on, and kubectl prints the status.

I don't have an example output here as their were no terminating pods from my run.

Reply:
OKAY